spec/spec_helper.rb

```diff
@@ -25,5 +25,8 @@
   config.include Rack::Test::Methods
   config.raise_errors_for_deprecations!
 
+  config.filter_run :focus
```

Comment:
Mmm... maybe it will be useful? I often use `focus: true` in specs when debugging. If you disagree I'll remove this.